docs/root/intro/arch_overview/listeners/connection_limiting.rst

```diff
@@ -0,0 +1,7 @@
+.. _arch_overview_connection_limit:
+
+Connection limiting
+===================
+
+Envoy supports local (non-distributed) connection limiting of L4 connections via the
```

Comment:
Can you also reference/document the runtime listener based connection limits here also?

Reply:
Reference this doc in the runtime listener: https://www.envoyproxy.io/docs/envoy/latest/configuration/listeners/runtime

Reply:
I commented there, but please move that here. The idea is that this is the arch overview documentation for all types of connection limiting.